ios/RNSScreenStack.mm

```diff
@@ -646,6 +646,17 @@ - (void)layoutSubviews
 {
   [super layoutSubviews];
   _controller.view.frame = self.bounds;
+
+  // We need to update the bounds of the modal views here, since
+  // for contained modals they are not updated by modals themselves.
+  for (UIViewController *modal in _presentedModals) {
+    BOOL isModalBeingDismissed = [modal isKindOfClass:[RNSScreen class]] && ((RNSScreen *)modal).isBeingDismissed;
```

Comment:
Could you add a comment why we are not taking modals being dismissed into account and why taking the modal's view's superview frame works correctly? Other than that, LGTM 🚀

Reply:
Sure!

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/2011/commits/40f4f32a411fa7f44d97c4edcd5bdf57769ddde0.